tests/src/model/pages/image-edit-page.ts

```diff
@@ -33,4 +35,18 @@ export class ImageEditPage extends BasePage {
     this.name = name;
     this.imageTag = page.getByLabel('imageTag');
   }
+
+  async renameImage(name: string): Promise<ImagesPage> {
```

Comment:
I am looking at the page/dialog dom: 
```
<div class="modal svelte-ipux5s" role="dialog" aria-label="Edit Image" aria-modal="true">
```

We can easily improve the edit-image to use this root element as a container locator. Can yu please fill an issue that would follow up to this? no need to tackle that in this PR.

Reply:
Created issue here: https://github.com/containers/podman-desktop/issues/5752

Reply:
thx